### ISSUE TITLE
Broken result video in any width more than 80

### ISSUE THREAD

**Issue by normaliok** [TheScienceElf/Video-to-Text#8]
When I change width value in convert.py to any value not equal 80 the result video looks broken

**Comment by CatKasha** [comment]
Did you use convert.py in Windows? Maybe width of command prompt / terminal is lower or not equal to width of video

**Comment by normaliok** [comment]
> Did you use convert.py in Windows? Maybe width of command prompt / terminal is lower or not equal to width of video

No, I'm using Ubuntu and make an exe file for windows. I didn't mean video looks like a noise of characters in console window - of course I have to adjust window size to get a correct image. But when I try to use a different width (not 80) generated video is really short and has no any correct frames

**Comment by CatKasha** [comment]
Ah, i saw this issue. Did you match fps and width parameters in both convert.py and playback.c?
To calculate HEIGHT in playback.c you need calculate it via this (width / (2 * (16 / 9))) and round down result number

**Comment by normaliok** [comment]
> Ah, i saw this issue. Did you match fps and width parameters in both convert.py and playback.c? To calculate HEIGHT in playback.c you need calculate it via this (width / (2 * (16 / 9))) and round result number down

But HEIGHT is commented in playback.c   Should I uncomment it first?

**Comment by CatKasha** [comment]
`#define` in C lang is not a comment, compiler will replace all `HEIGHT` with that number in the code when you compile your program

**Comment by normaliok** [comment]
> `#define` in C lang is not a comment, compiler will replace all `HEIGHT` with that number in the code when you compile your program

Just have tested - no success. What is correct height for width=120? 34?

**Comment by CatKasha** [comment]
(120 / (2 * (16 / 9))) = 33.75, round down, 33

**Comment by normaliok** [comment]
> (120 / (2 * (16 / 9))) = 33.75, round down, 33

Nice! It works now!

**Comment by VivIsBee** [comment]
> > (120 / (2 * (16 / 9))) = 33.75, round down, 33
> 
> Nice! It works now!

close the issue then
